LabBuilder/LabBuilder.psm1

```diff
@@ -798,8 +798,8 @@ function Download-LabResources {
     # Downloading Lab Resources
     Write-Verbose -Message $($LocalizedData.DownloadingLabResourcesMessage)
 
-    # Bootstrap Nuget
-    $null = Get-PackageProvider -Name NuGet -ForceBootstrap -Force
+    # Bootstrap Nuget # This needs to be a test, not a force 
+    # $null = Get-PackageProvider -Name NuGet -ForceBootstrap -Force
```

Comment:
Can you clarify what you mean here? This is some code we often integrate up on AppVeyor and for unit tests to ensure that the the Nuget Anycpu-Nuget.exe has been downloaded.